CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ All notable changes to this project will be documented in this file.
 -   #1691 : Add Python support for list method `clear()`
 -   #1575 : Add support for homogeneous tuple type annotations on variables.
 -   #1425 : Add support for `numpy.isnan`, `numpy.isinf` and `numpy.isfinite`.
--   #1738 : Add Python support for set method `add()`
+-   #1738 : Add Python support for scalar sets with `{}` and `add()` method.
```

Comment:
Can we split into two points? One about constructing sets, the other about the `add` method.

Reply:
Yes, but you have a typo: list -> set

Reply:
Sorry, I will resolve it

Reply:
fixed : 61bb8ff